simplecaptcha cannot be found in the Maven repository

When trying to deploy a server the log states:

Downloading: http://repo1.maven.org/maven2/nl/captcha/simplecaptcha/1.2.1/simplecaptcha-1.2.1.pom
Downloading: http://repo1.maven.org/maven2/nl/captcha/simplecaptcha/1.2.1/simplecaptcha-1.2.1.pom
Downloading: http://repo1.maven.org/maven2/nl/captcha/simplecaptcha/1.2.1/simplecaptcha-1.2.1.pom
Downloading: http://repo1.maven.org/maven2/nl/captcha/simplecaptcha/1.2.1/simplecaptcha-1.2.1.jar
Downloading: http://repo1.maven.org/maven2/nl/captcha/simplecaptcha/1.2.1/simplecaptcha-1.2.1.jar
[ERROR] Artifact: nl.captcha:simplecaptcha:jar:1.2.1 has no file.
org.apache.maven.artifact.resolver.ArtifactNotFoundException: Could not find artifact nl.captcha:simplecaptcha:jar:1.2.1 in central (http://repo1.maven.org/maven2)

Try downloading the file manually from the project website.

Then, install it using the command: 
    mvn install:install-file -DgroupId=nl.captcha -DartifactId=simplecaptcha -Dversion=1.2.1 -Dpackaging=jar -Dfile=/path/to/file

Alternatively, if you host your own repository you can deploy the file there: 
    mvn deploy:deploy-file -DgroupId=nl.captcha -DartifactId=simplecaptcha -Dversion=1.2.1 -Dpackaging=jar -Dfile=/path/to/file -Durl=[url] -DrepositoryId=[id]

  nl.captcha:simplecaptcha:jar:1.2.1

--- a/pom.xml
+++ b/pom.xml
@@ -334,6 +334,19 @@
 							</pluginExecutions>
 						</lifecycleMappingMetadata>
 					</configuration>
+				</plugin>
+				<!--  integration test plugin -->
+				<plugin>
+					<artifactId>maven-failsafe-plugin</artifactId>
+					<version>2.6</version>
+					<executions>
+						<execution>
+							<goals>
+								<goal>integration-test</goal>
+								<goal>verify</goal>
+							</goals>
+						</execution>
+					</executions>
 				</plugin>	
 			</plugins>
 		</pluginManagement>

